### ISSUE TITLE
SC2086: false positive while passing parameters in variable

### ISSUE THREAD

**Issue by volans-** [koalaman/shellcheck#267]
``` bash
PARAMS="-a -b -c 'test'"

program ${PARAMS}
```

In this case double quoting will prevent to interpret the parameters and the whole PARAMS string will be passed as the first parameter.
